diff --git a/data/open-source projects/2-spring-cloud-zuul-ratelimit/2.8/2-8-b.java b/data/open-source projects/2-spring-cloud-zuul-ratelimit/2.8/2-8-a.java
index 18da494..4412e46 100644
--- a/data/open-source projects/2-spring-cloud-zuul-ratelimit/2.8/2-8-b.java	
+++ b/data/open-source projects/2-spring-cloud-zuul-ratelimit/2.8/2-8-a.java	
@@ -48,7 +48,10 @@ import io.github.bucket4j.grid.jcache.JCache;
 import org.apache.ignite.IgniteCache;
 import org.infinispan.functional.FunctionalMap.ReadWriteMap;
 import org.springframework.beans.factory.annotation.Qualifier;
-import org.springframework.boot.autoconfigure.condition.*;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingClass;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.boot.autoconfigure.domain.EntityScan;
 import org.springframework.boot.context.properties.ConfigurationPropertiesBinding;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
@@ -72,7 +75,6 @@ import static com.marcosbarbero.cloud.autoconfigure.zuul.ratelimit.config.proper
  */
 @Configuration
 @EnableConfigurationProperties(RateLimitProperties.class)
-@ConditionalOnBean(RouteLocator.class)
 @ConditionalOnProperty(prefix = PREFIX, name = "enabled", havingValue = "true")
 public class RateLimitAutoConfiguration {
 
