diff --git a/data/open-source projects/2-spring-cloud-zuul-ratelimit/2.1/2-1-b.java b/data/open-source projects/2-spring-cloud-zuul-ratelimit/2.1/2-1-a.java
index 3836ec4..68f9ae9 100644
--- a/data/open-source projects/2-spring-cloud-zuul-ratelimit/2.1/2-1-b.java	
+++ b/data/open-source projects/2-spring-cloud-zuul-ratelimit/2.1/2-1-a.java	
@@ -16,6 +16,7 @@
 
 package com.marcosbarbero.cloud.autoconfigure.zuul.ratelimit.filters;
 
+import static com.marcosbarbero.cloud.autoconfigure.zuul.ratelimit.config.properties.ResponseHeadersVerbosity.NONE;
 import static com.marcosbarbero.cloud.autoconfigure.zuul.ratelimit.config.properties.ResponseHeadersVerbosity.VERBOSE;
 import static com.marcosbarbero.cloud.autoconfigure.zuul.ratelimit.support.RateLimitConstants.HEADER_LIMIT;
 import static com.marcosbarbero.cloud.autoconfigure.zuul.ratelimit.support.RateLimitConstants.HEADER_QUOTA;
@@ -107,7 +108,7 @@ public class RateLimitPreFilter extends AbstractRateLimitFilter {
 
             responseHeaders.put(HEADER_RESET, String.valueOf(rate.getReset()));
 
-            if (VERBOSE.equals(properties.getResponseHeaders())) {
+            if (!NONE.equals(properties.getResponseHeaders())) {
                 final String httpHeaderKey = key.replaceAll("[^A-Za-z0-9-.]", "_").replaceAll("__", "_");
                 for (Map.Entry<String, String> headersEntry : responseHeaders.entrySet()) {
                     String header = VERBOSE.equals(properties.getResponseHeaders()) ? headersEntry.getKey() + "-" + httpHeaderKey : headersEntry.getKey();
